api/task_processor/models.py

```diff
@@ -90,12 +101,14 @@ class Meta:
     def create(
         cls,
         task_identifier: str,
+        priority: TaskPriority = TaskPriority.NORMAL,
```

Comment:
@matthewelwell think we can get rid of `schedule_task` and just use this method directly?

Reply:
We _can_, but I quite like having the helper method to differentiate them. What's the benefit in removing it?

Reply:
I think the name is slightly odd, no? I'd expect it to schedule the task(i.e: put on some queue?) but it only creates the class instance. Does that make sense?

Reply:
I see, you mean that it doesn't actually persist it to the DB? But that's the same as the `create` method too, right? We could update the name to `create_scheduled_task` ? Maybe you're right though that we should consolidate the 2 methods.

Reply:
> you mean that it doesn't actually persist it to the DB?
Yes, exactly

Reply:
I've removed schedule_task method but I am not sure if it looks any better :confused: ?

Reply:
Well, there's less code so that's always a good thing. I think I would prefer to keep the logic regarding the queue in the Task model though. Can we just handle that in the `create` method now instead?

Reply:
yeah, done